src/main/java/uk/gov/ons/census/casesvc/service/EventProcessor.java

```diff
@@ -28,15 +24,15 @@ public void processSampleReceivedMessage(CreateCaseSample createCaseSample) {
     int questionnaireType =
         QuestionnaireTypeHelper.calculateQuestionnaireType(caze.getTreatmentCode());
     UacQidLink uacQidLink = uacProcessor.saveUacQidLink(caze, questionnaireType);
-    uacProcessor.emitUacUpdatedEvent(uacQidLink, caze);
+    uacProcessor.emitUacUpdatedEvent(uacQidLink, caze, true);
     caseProcessor.emitCaseCreatedEvent(caze);
-    uacProcessor.logEvent(uacQidLink, CASE_CREATED_EVENT_DESCRIPTION);
-    uacProcessor.logEvent(uacQidLink, UAC_QID_LINKED_EVENT_DESCRIPTION);
+    uacProcessor.logEvent(uacQidLink, CASE_CREATED_EVENT_DESCRIPTION, null);
```

Comment:
can't you overload the `logEvent` method so that it only takes 2 parameters and defaults the third to null?